components/views/dashboard/index.tsx

```diff
@@ -96,7 +97,7 @@ const Dashboard = () => {
                 <UsersOverview />
             </div>
             <div className={styles.productivityCharts}>
-                <div>DEVELOPERS HOURS LINE CHART</div>
+                <LineChart />
```

Comment:
I like your thinking on making the chart a component, but if we're gonna do this (which we should), let's go the full mile instead of half way. This shouldn't be called a `LineChart`. It should be named appropriately since it's going to be handling the direct data of the specific chart.

My suggestion is to
1. Have a folder called `line-charts`, index in this folder be the `ResponsiveLineChart` component, that takes in the data, and other potentially reusable props that relate to the chart when we need them. 
2. Create a file in the same folder called `DeveloperHoursLineChart`, have that component import `LineChart`, and do all of the set up calculations needed to create the appropriate data, then feed it into `LineChart`
3. Import the `DeveloperHoursLineChart` to `index.tsx` file at the base of the `/charts/index.tsx` folder/path, and export as a named variable. Example of what the new index file should look like (once you've moved everything around)

```
import DeveloperHoursLineChart from './line-charts/DeveloperHoursLineChart)

export { DeveloperHoursLineChart }
```

This way, we can follow this template for the pie chart, and any other charts we will almost certainly be adding at some point along the journey. Now they're separated, concise and specific to each chart type, but can all be imported from the exact same import location cleanly.